code/game/objects/structures/displaycase.dm

```diff
@@ -256,6 +256,9 @@ var/global/list/captain_display_cases = list()
 			new /obj/machinery/constructable_frame/machine_frame(T)
 		qdel(src)
 		return
+	if(istype(W,/obj/item/weapon/grab))

```

Comment:
`if(!user.drop_item())` does not work, in testing. How does the ABSTRACT check have to be set up?